packages/components/pie-button/src/button.scss

```diff
@@ -38,23 +46,68 @@
     line-height: 28px;
 }
 
-.o-btn--primary {
+.o-btn--primary:not(.is-disabled) {
     background-color: dt.$color-interactive-brand;
     color: dt.$color-content-interactive-primary;
+
+    &:hover {
+        background-color: darken(dt.$color-interactive-brand, dt.$color-hover-01);
+    }
+
+    &:active {
+        background-color: darken(dt.$color-interactive-brand, dt.$color-active-01);
+    }
 }
 
-.o-btn--secondary {
+.o-btn--secondary:not(.is-disabled) {
     background-color: dt.$color-interactive-secondary;
     color: dt.$color-content-interactive-secondary;
+
+    &:hover {
+        background-color: darken(dt.$color-interactive-secondary, dt.$color-hover-01);
+    }
+
+    &:active {
+        background-color: darken(dt.$color-interactive-secondary, dt.$color-active-01);
+    }
 }
 
-.o-btn--outline {
-    border: 1px solid dt.$color-border-strong;
+.o-btn--outline:not(.is-disabled) {
     background-color: dt.$color-container-default;
     color: dt.$color-content-interactive-tertiary;
+    outline: 1px solid dt.$color-border-strong;
+
+    &:hover {
+        background-color: darken(dt.$color-container-default, dt.$color-hover-01);
+    }
+
+    &:active {
+        background-color: darken(dt.$color-container-default, dt.$color-active-01);
+    }
 }
 
-.o-btn--ghost {
+.o-btn--ghost:not(.is-disabled) {
     background-color: dt.$color-container-default;
     color: dt.$color-content-link;
+
+    &:hover {
+        background-color: darken(dt.$color-container-default, dt.$color-hover-01);
+    }
+
+    &:active {
+        background-color: darken(dt.$color-container-default, dt.$color-active-01);
+    }
 }
+
+.is-disabled {
```

Comment:
Pretty sure the answer is no, but can this class affect elements outside of the web component in the DOM?

Reply:
is there any reason not to use this as a modifier class i.e. `o-btn--is-disabled`?

Reply:
I think styles are scoped using shadow DOM in lit, but I'm happy to update the class name to `o-btn--is-disabled` for consistency with modes.